### PR TITLE
sample: preview toggle

### PR DESCRIPTION
**Pull request by minh-rakuten**
# Description
Demo app:
Move the preview toggle below build info section.
Restore miniapp navigator.

## Links
MINI-1030

# Checklist
- [x] I have read the [contributing guidelines](../CONTRIBUTING.md).
- [ ] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `./gradlew check` without errors

### CODE REVIEW

File: testapp/src/main/res/layout/settings_menu_activity.xml [hunk 1]
```diff
@@ -23,32 +23,27 @@
...
-            android:id="@+id/textInfo"
+        <com.google.android.material.switchmaterial.SwitchMaterial
+            android:id="@+id/switchPreviewMode"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:layout_margin="@dimen/small_8"
             android:padding="@dimen/small_8"
-            android:textColor="@android:color/black"
-            tools:text="Build Info" />
+            android:text="@string/lb_preview_mode" />
```
Comment (khairul-alam-licon) [inline testapp/src/main/res/layout/settings_menu_activity.xml:46]: I wish we could also add the following lines for this `SwitchMaterial` to highlight the UI.

```
android:textSize="@dimen/text_large_16"
android:textColor="@android:color/black" 
```

**Review by khairul-alam-licon — APPROVED** [review]
Nice, Thanks 👍

File: testapp/src/main/res/layout/settings_menu_activity.xml (hunk 1 above)
Comment (minh-rakuten) [reply testapp/src/main/res/layout/settings_menu_activity.xml:46]: I prefer our current one.

**Review by corycaywood — APPROVED** [review]
Looks good 👍

**Comment by minh-rakuten** [comment]
I'm sorry that I forget to set back miniAppNavigator in demo app. I have restored it.

**Pull request merged**